readthedocs/comments/models.py

```diff
@@ -111,6 +111,7 @@ class DocumentNodeSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = DocumentNode
+        exclude = ('')
```

Comment:
Yea, but that was giving me an error when I was extending the fields list with custom fields on the Serializer.